pytorch_lightning/trainer/supporters.py

```diff
@@ -172,12 +177,10 @@ class CycleIterator(object):
 
     def __init__(self, loader: Any, length: Optional[int] = None):
         """
-
         Args:
             loader: the loader to restart for cyclic (and optionally infinite) sampling
             length: the number of batches to sample (with restarted loaders if necessary) before raising StopIteration
                 if None: infinite
-
```

Comment:
I will add back these removed lines
so docs get rendered correctly (an empty new line is needed for rendering this correctly in html)

Reply:
Planning to do this on this PR?